index.js

```diff
@@ -0,0 +1,84 @@
+const title = document.querySelector('#title');
+const author = document.querySelector('#author');
+const add = document.querySelector('#add');
+const bookList = document.querySelector('#book-list');
+
+let list = JSON.parse(localStorage.getItem('bookList')) || [
+  {
+    title: 'Martin Fierro',
+    author: 'José Hernandez',
+  },
+  {
+    title: 'Harry Potter and the Chamber of Secrets',
+    author: 'J. K. Rowling',
+  },
+];
+
+function saveToLocalStorage() {
+  localStorage.setItem('bookList', JSON.stringify(list));
+}
+
+let htmlList = '';
+
+function renderBooks() {
+  htmlList = '';
+
+  list.forEach((book) => {
+    htmlList += `
+    <div>
+      <p>
+        <span class="title">${book.title}</span>
+        <br>
+        ${book.author}
+        <br>
+        <button class="remove">Remove</button>
+      </p>
+      <hr>
+    </div>`;
+  });
+
+  bookList.innerHTML = htmlList;
+
+  const removeButtons = document.querySelectorAll('.remove');
+  removeButtons.forEach((button) => {
+    // eslint-disable-next-line
```

Comment:
You have done a perfect work, however, we have a few points of improvement, talking about linter, it helps us clean and standardize our code regarding best practices and DRY method, to say so, I believe it is not a good practice to disable it in our code, I would strongly recommend you fix it up and remove the disabling lines as this will contribute to your professionalism and gain some good ways of writing a clean code in the near future, what do you think about it? 😃